qute-keepassxc

```diff
@@ -27,6 +27,9 @@ def parse_args():
                         help='GPG key to encrypt KeepassXC auth key with')
     parser.add_argument('--insecure', action='store_true',
                         help="Do not encrypt auth key")
+    parser.add_argument('--connection', '-c', default='keepassxc.key',
+                        help='basename of the KeepassXC connection file')
```

Comment:
I like the feature, just not the name.

`connection` is too generic here, I don't want to burn that name. It actually is the `associationkeyfilename` which, admittedly, isn't a very handy name, but still better than `connection`.
What do you think?